### DISCUSSION TITLE
Request a new release w/ Linux Kernel 6.6 for Milk-V Meles

### DISCUSSION THREAD

**Discussion by lazyparser** [revyos/revyos#84] [General]
According to this [Chinese posts](https://mp.weixin.qq.com/s/71lFCsy2i-rOpw9FbXg8RA) the Milk-V Meles are now fully supported and the community is co-maintained by the RuyiSDK Project. I'd like to wish a new kernel for fun and convenience. Currently it has a 5.10.x
 kernel, which is somewhat out of date.

**Comment by FIFCC** [comment]
The request has been received and there are plans to release a new kernel. It is expected that the image using the new kernel will be released within this month.

**Comment by FIFCC** [comment]
> According to this [Chinese posts](https://mp.weixin.qq.com/s/71lFCsy2i-rOpw9FbXg8RA) the Milk-V Meles are now fully supported and the community is co-maintained by the RuyiSDK Project. I'd like to wish a new kernel for fun and convenience. Currently it has a 5.10.x kernel, which is somewhat out of date.

New images supporting the 6.6 kernel are now available.

https://mirror.iscas.ac.cn/revyos/extra/images/meles/20241229/
